### PR TITLE
Remove criteria from Forms' classes

### PR DESCRIPTION
**Pull request by Beacze**
Depends on #945

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java [hunk 1]
```diff
@@ -455,8 +455,20 @@ public void refresh(Process process) {
      * @return list of JSON objects with processes for specific process id
      */
     public List<JSONObject> findByProjectId(Integer id) throws DataException {
-        QueryBuilder query = createSimpleQuery("project", id, true);
-        return searcher.findDocuments(query.toString());
+        return searcher.findDocuments(getQueryProjectId(id).toString());
+    }
+
+    public Long findCountForSortHelperImages(Integer projectId) throws DataException {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java]: Public method without JavaDoc. Please add the missing JavaDoc!

File: Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java [hunk 2]
```diff
@@ -455,8 +455,20 @@ public void refresh(Process process) {
...
+        return findCountAggregation(getQueryProjectId(projectId).toString(), "sortHelperImages");
+    }
+
+    public Double findSumForSortHelperImages(Integer projectId) throws DataException {
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java]: Public method without JavaDoc. Please add the missing JavaDoc!

File: Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java]: done

File: Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java (hunk 2 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java]: done

File: Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/UserDAO.java [hunk 3]
```diff
@@ -127,4 +127,13 @@ public Long count(String query) throws DAOException {
...
         refreshObject(user);
     }
+
+    /**
+     * Get all active users sorted by surname and name.
+     *
+     * @return sorted list of all active users as User objects
+     */
+    public List<User> getAllActiveUsersSortedByNameAndSurname() {
+        return search("FROM User WHERE active = 1 AND visible IS NULL ORDER BY surname ASC, name ASC");
```
Comment (henning-gerhardt) [inline Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/UserDAO.java]: Is `visible` not changed to an integer value and renamed to  `deleted` and `NULL` should not possible any more?

Comment (Beacze) [reply Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/UserDAO.java]: true

**Comment by henning-gerhardt** [comment]
There is a merge conflict in `Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java` - please fix this merge conflict.

**Pull request merged**